src/components/VerificationTracker/VerifyEmail.js

```diff
@@ -19,24 +20,37 @@ const VerifyEmail = () => {
       setError(e);
     }
   };
+  const idemUrl = `${process.env.REACT_APP_IDEM_URL}?firstName=${userDetails.firstName}&lastName=${userDetails.lastname}&email=${userDetails.email}&userId=${userDetails.userID}`;
   return (
     <div>
-      <p>
-        <b>Verify your email to continue</b>
-      </p>
-      <ErrorMessage error={error} />
-      <p>
-        We have sent you a link to verify your email. If you do not see an email
-        from us please check your spam folder.
-      </p>
-      <SubmitSpinnerButton
-        isSubmitting={isSending}
-        variant="primary"
-        submitText="Resend verification email"
-        block={false}
-        onClick={onResendClick}
-        icon={hasResent && !error && "checkmark-circle"}
-      />
+      <div className="d-flex justify-content-between align-items-center">
+        <div>
+          <p>
+            <b>Verify your email to continue</b>
+          </p>
+          <ErrorMessage error={error} />
+          <p>
+            We have sent you a link to verify your email. <br></br>
+            If you do not see an email from us please check your spam folder.
+          </p>
+          <SubmitSpinnerButton
+            isSubmitting={isSending}
+            variant="primary"
+            submitText="Resend verification email"
+            block={false}
+            onClick={onResendClick}
+            icon={hasResent && !error && "checkmark-circle"}
+          />
+        </div>
+        <div>
+          <p className="ml-3">
+            <b>Use Idem App to Verfiy</b>
+          </p>
+          <div className="p-2">
+            <QRCode value={idemUrl ? idemUrl : null} size="200" />
```

Comment:
Does it make sense to remove the QRCode component from the DOM if there isnt a idemURL? What does an empty QRCode look like?